pkg/utils/blake.go

```diff
@@ -0,0 +1,133 @@
+package utils
+
+func IV() [8]uint32 {
```

Comment:
why do you use uint32 for all the types in this file? Is it impossible for the values to exceed 2**32?

Reply:
yes the initialization vector is supposed to be a constant, [8]uint32 is correct. SIGMA can be uint8 though, I've fixed.

Reply:
okok, good job 🚀